### PR TITLE
DEPLOY-632: Update documentation where ingressvalues.yaml file is cre…

### PR DESCRIPTION
**Pull request by rpopa**
…ated

   - updated README.md

### CODE REVIEW

**Comment by rpopa** [comment]
Check this: https://github.com/Alfresco/alfresco-infrastructure-deployment/tree/DEPLOY-632#nginx-ingress-custom-configuration

File: README.md [hunk 1]
```diff
@@ -93,27 +93,58 @@ By default, this chart deploys the [nginx-ingress chart](https://github.com/kube
...
+rbac:
+  create: true
+controller:
+  config:
+    ssl-redirect: "false"
+  scope:
+    enabled: true
+EOF
+
+helm install stable/nginx-ingress --version=0.14.0 -f ingressvalues.yaml \
```
Comment (rgauss) [inline README.md]: Do we want to recommend installing the ingress directly at all?

Should we only document settings for infrastructure values?

File: README.md [hunk 2]
```diff
@@ -147,20 +180,46 @@ helm install alfresco-incubator/alfresco-infrastructure \
 -f infravalues.yaml \
 --namespace $DESIREDNAMESPACE
 ```
+</p>
+</details>
+
+<details>
+<summary>settings for deploy the nginx-ingress chart</summary>
```
Comment (rgauss) [inline README.md]: Here as well, do we want / need this section?

File: README.md [hunk 3]
```diff
@@ -217,10 +282,54 @@ helm install alfresco-incubator/alfresco-infrastructure \
 --namespace $DESIREDNAMESPACE
 ```
 
+</p>
+</details>
+
+
+<details>
+<summary>deploy the nginx-ingress chart</summary>
```
Comment (rgauss) [inline README.md]: Here as well, do we want / need this section?

**Review by rgauss — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rpopa) [reply README.md]: Well, the deployment of alfresco-infrastructure starting version 3 contains indeed as dependency alfresco-identity-service and nginx-ingress. But for version 1.x and 2.x it does not. Second, on README for alfresco-identity-service, we provide steps to deploy the chart, and the deployment needs the nginx-ingress chart to be deployed before. Suggesting to deploy the alfresco-infrastructure without alfresco-identity-service in order to have an ingress deployment that we will use it for the identity chart deployment, it creates confusion.

What do you think is the most suitable approach in this case? Remove the steps for deploying the ingress directly and update the alfresco-identity service README to use the alfresco-infrastructure (deploy without alfresco-identity-service or with)?

**Review by rgauss — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -241,11 +245,21 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
...
+    alfresco-incubator/alfresco-infrastructure \
 ```
 
 Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
 
 ```console
 $ helm install alfresco-incubator/alfresco-infrastructure --name my-release -f values.yaml
 ```
+
+You have also the posibility to deploy only the ingress chart or any other component by itself if needed, using the following:
```
Comment (rpopa) [inline README.md]: Not sure about this section if I should add it here. I will remove it if is not ok

Comment (jottley) [reply README.md]: Should we add the "component.enabled" parameters to the configuration parameter chart above? Then this section would possibly not be needed.

Comment (rpopa) [reply README.md]: Yes, that will clear up my doubts regarding the options on how to deploy ingress or alfresco-identity-service.
I will do that!

**Review by jottley — APPROVED** [review]

**Pull request merged**